client/src/components/editor/LinkSource.js

```diff
@@ -55,7 +55,7 @@ class LinkSource extends Component {
         const contentState = editorState.getCurrentContent()
         const contentStateWithEntity = contentState.createEntity(
           entityType.type,
-          "MUTABLE",
+          "IMMUTABLE",
```

Comment:
It makes sense to make ANET links immutable ( https://www.draftail.org/docs/content-storage#entities ), however, it would appear I am now no longer able to create abritrary (URL) links (except by pasting a URL copied from sonewhere else, which I then can no longer edit).

Reply:
In the previous implementation you were only allowed to edit links through a menu (when the link was clicked on in the editor). However, since that is gone I guess we should make them mutable, so we can edit them without copy/pasting.

Reply:
On `candidate`, I can edit the text *and* the corresponding link independently.

Reply:
For ANET links, it does not make sense to be able to edit the text, since it should be derived from the link (by whatever LinkTo comes up with). But for plain URL's, I'd like the old behaviour. That would mean deciding what the type of link is and do different things with it (for starters, one would be immutable, the other mutable). Would it be possible to add a new option/button to the pop-up modal dialog for abitrary URL's?

Reply:
@VassilIordanov commented the following on this issue: 

> Jose, I do not think we should have external links (at least I do not want to encourage the users of doing that)

The editor does not support 2 types of custom links, so that is why the popover link editor is gone (but not removed from the code, see `LinkSource.js` and `Link.js`). I think we should remove link support entirely for non ANET entities, as it is confusing and undesired. In the current version (`candidate` branch) when you type out a link it does not parse it as such.

Reply:
OK, so only ANET links then. About obsolete files, see https://github.com/NCI-Agency/anet/pull/2790#issuecomment-585617187 .

Reply:
I dropped all support for links not pointing to a ANET entity, but you can now paste a ANET entity link in the editor and it will be parsed to a correct `LintkTo` component on save.